mission_list.md

```diff
@@ -0,0 +1,60 @@
+>It can be updated through an on-chain 2/3-majority rank-weighted vote across all Members.
+>
+>Some obvious cornerstones that must be upheld by all projects are:
+>
+>- Must be FOSS (Free and Open-Source Software).
+>- Source code must be accessible without a login requirement.
+>- All members of PoToC must have permission to propose changes and raise issues.
+>- The Mission List will be defined as part of the initial seeding.
+>
+>Core Tooling
+>
+>Generally, all tools that are used by The Fellowship to craft, test, and validate updates fall into this category. But only the tools that are predominantly developed for this cause shall be maintained by PoToC. For example, the Rust language is not predominantly developed for Polkadot; hence, it does not fit here.
+>
+>dApp Tooling
+>
+>This category contains all tools and libraries that are used to build on Polkadot and are predominantly developed for this cause.
+
+(This is my personal opinion on a 
+
+## DApp Tooling
+
+- [Polkadot-JS](https://github.com/polkadot-js/apps)
+- [PAPI](https://www.npmjs.com/package/@polkadot-api/substrate-client)
+- [Chopsticks](https://github.com/AcalaNetwork/chopsticks)
+- [Zombienet](https://github.com/paritytech/zombienet) + [v2](https://github.com/paritytech/zombienet-sdk)
+- [Substrate Connect](https://github.com/paritytech/substrate-connect)
+- [Scale TS](https://npmjs.com/package/scale-ts)
+
+## Core Tooling
+
+- [SubXt](https://github.com/paritytech/subxt)
+- [Subway](https://github.com/AcalaNetwork/subway)
+- [Zepter](https://github.com/ggwpez/zepter)
+- [Substrate Weight Compare](https://github.com/ggwpez/substrate-weight-compare)
+- [OpenGov CLI](https://github.com/joepetrowski/opengov-cli)
+- [Sidecar](https://github.com/paritytech/substrate-api-sidecar)
+- [scale-*](https://github.com/paritytech/scale-decode), [more](https://github.com/polkadot-tooling-collective/collective/blob/dd96bc056dfa47f0bc6a885c7afd25ed78345a11/join_request/jameswilson.yml#L26)
+- [srtool](https://github.com/paritytech/srtool) (maintainer did not apply yet)
+- [subwasm](https://github.com/chevdor/subwasm) (ditto)
+- [virto SDK/libwallet](https://github.com/virto-network/virto-sdk/tree/main/libwallet)
+- [virto SDK/scales](https://github.com/virto-network/virto-sdk/tree/main/scales)
+
+## Undecided
+
+- [virto SDK/sube](https://github.com/virto-network/virto-sdk/tree/main/sube)
+- [polkadot-JS extension](https://github.com/polkadot-js/extension)
+- [cypress polkadot wallet](https://github.com/ChainSafe/cypress-polkadot-wallet)
+- [Zondax Ledger](https://github.com/zondax/ledger-polkadot)
+- [Rust Test](https://github.com/ltfschoen/RustTest)
+- [Polkadot JS test](https://github.com/ltfschoen/PolkadotJSTest)
+- [NFT coretime](https://github.com/ltfschoen/nft-coretime)
+- [Flappy tips](https://github.com/ltfschoen/flappytips)
+- [XCM Template](https://github.com/ltfschoen/XCMTemplate)
```

Comment:
This looks promising @ltfschoen, do you know if it used by others / has any adoption?

Reply:
i shared it on https://substrate.stackexchange.com/ once but i had a bug in it at the time that i've since fixed.
i haven't marketed it at all, i only shared it incase it'd help someone else.
if you have any feedback or suggestions i'm willing to take them onboard to try and turn it into something more useful, or integrate it into something else that exists

Reply:
Okay i included it for now.